components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineSession.kt

```diff
@@ -272,6 +273,11 @@ class SystemEngineSession(
         webSettings.savePassword = false
     }
 
+    private fun setUseWideViewPort(s: WebSettings, useWideViewPort: Boolean?) {
+        this@SystemEngineSession.useWideViewPort = useWideViewPort
```

Comment:
nit: I can't see the whole code in this review mode - but can't that be just `this.useWideViewPort = useWideViewPort`?